pyproject.toml

```diff
@@ -115,7 +115,9 @@ filterwarnings = [
   'ignore:.*You are using pyarrow version',
   'ignore:.*but when imported by',
   'ignore:Distributing .*This may take some time',
-  'ignore:.*The default coalesce behavior'
+  'ignore:.*The default coalesce behavior',
+  "ignore:.*The 'downcast' keyword in fillna is deprecated",
+  'ignore:.*Falling back on a non-pyarrow code path which',
```

Comment:
I'm a bit hesitant about adding these here (in fact, we've already got too many we're ignoring broadly - mostly my fault here 🙈 ) - would it be possible to filter these just in the affected test(s)?

Reply:
Yes. It will better in that way c: